Makefile

```diff
@@ -103,13 +103,23 @@ tidy:
 
 $(PLATFORMS): genproto
 ifeq ($(OS),Windows_NT)
-	set GOOS=$(os)&&set GOARCH=amd64&&go build ${LDFLAGS} -o $(CURR_DIR)/$(BINARY)
+	set GOOS=$(os)&&set GOARCH=amd64&&go build ${LDFLAGS} -o $(CURR_DIR)/$(BINARY).exe
 else
 	GOOS=$(os) GOARCH=amd64 go build ${LDFLAGS} -o $(CURR_DIR)/$(BINARY)
 endif
 .PHONY: $(PLATFORMS)
 
 
+docker-local-build: genproto
+	GOOS=linux GOARCH=amd64 go build ${LDFLAGS} -o $(BIN_DIR)/$(BINARY)
+	cd cmd/hare ; GOOS=linux GOARCH=amd64 go build -o $(BIN_DIR)/go-hare; cd ..
```

Comment:
You can avoid all the `cd ..` here since, IIRC, make runs each line of the Makefile in its own shell. I'd probably rewrite as `cd $TARGET_DIR && ...`